grant-project/grant-monorepo
# Fix regressions from Readable URLs #190

Closes #191. Closes #192.
Related to #190.

### Fixes
- ensure `fetchProposal` is called with `proposalId`
- display `proposalAddress` instead of `proposalId` on `ProposalCard`
- update admin to use `proposalAddress`

### Testing
UI
- browse proposals, ensure that the proposalAddress is displayed

Ensure the following actions behave:
- create proposal and cancel it
- create proposal, fund it, vote for refund and payback
- create proposal, fund it, request non-immediate MS and then vote against it

Admin
- should render proposals
- should be able to "refresh contract" successfully
